sequencer.c

```diff
@@ -990,6 +991,7 @@ static int run_git_commit(const char *defmsg,
 			  struct replay_opts *opts,
 			  unsigned int flags)
 {
+	char message[opts->content_len + opts->subject_len];
```

Comment:
I do not think that C allows variable-sized arrays on the stack. The CI build [agrees](https://github.com/git/git/runs/5570793178?check_suite_focus=true#step:5:235). You will need `xmalloc()` and `free()` for that.

Reply:
As suggested in the other comment I use *stdin* to transfer the commit message now.
So allocation is not needed anymore.